bundle-core/src/main/java/net/discdd/pathutils/TransportPaths.java

```diff
@@ -1,26 +1,38 @@
 package net.discdd.pathutils;
 
 import lombok.Getter;
-import lombok.Setter;
-
+import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.SEVERE;
 
 @Getter
-@Setter
 public class TransportPaths{
+    private static final Logger logger = Logger.getLogger(TransportPaths.class.getName());
+
     @Getter
-    public Path fromClient;
+    public final Path fromClient;
     @Getter
-    public Path fromServer;
+    public final Path fromServer;
     @Getter
-    public Path toClient;
+    public final Path toClient;
     @Getter
-    public Path toServer;
+    public final Path toServer;
 
     public TransportPaths(Path rootDir){
         this.toServer = rootDir.resolve("BundleTransmission/server");
```

Comment:
since toServer is the same as fromClient, you should pick only one and delete the other, otherwise bug fixes to one might not propagate to the other.